submissions/opeyemiPeter/database/transportation-company.sql

```diff
@@ -41,7 +41,43 @@ CREATE TABLE accidentCases(
  primary key(id),
  foreign key(passengerId) references registeredPassengers(id)
 );
-
+DROP TABLE accidentcases;
 SELECT * FROM accidentCases;
 
 
+-- Task 3 Transportation Data Query.
+
+# 1. Total number of passengers who survived
+	select count(*) from accidentCases where accidentStatus > 0;
+	-- Answers => 342
+
+# 2. total number of unsurvived passengers
+  SELECT COUNT(*) FROM accidentCases WHERE accidentStatus = 0;
+  -- Answers => 549
+
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+	SELECT fullName, sex from registeredPassengers
+	inner join tripDetails on 
+    registeredpassengers.id = tripDetails.passengerId 
+    where (age  < '27') and (embarkationPoint = 
+    ( "Q%"| "C%"));
+    
+# 4. Number of passengers who embarked at Southampton and survived
+	SELECT count(*) from accidentCases left join tripdetails on accidentcases.id = tripdetails.id where accidentStatus > 0 and embarkationPoint = 'S';
+    -- Answers => 218;
+
+# 5. 
+select registeredPassengers.id, registeredPassengers.fullName, COUNT(*) OVER () AS total_count
+from registeredPassengers 
+join tripDetails on registeredPassengers.id = tripDetails.id 
```

Comment:
Your query is wrong here. Kindly go through the query I figure out what you are not doing correctly

Reply:
Good evening @adaugochi , i have tried every possible way to query the database to arrive at a diffferent answer but it has always given either 7(whenever i use tripsdetail.id) or 0(when i use tripsdetail.passenger.Id) as a result. i have consulted other colleagues and within and outside the codecamp about this and I tried every solution provided by them still resulting to the same result. 
if these answers(0 & 7) are not correct, then i suppose the data database will be the problem. here is the result using the different ways to querying the table.
![image](https://user-images.githubusercontent.com/63324192/108126969-61cd9880-705f-11eb-9312-5760b28c133d.png)

Reply:
0 or 7 is not the answer. Though 7 is close to it. I have checked your query and the join statement is ok. Maybe some data are missing when importing. Kindly drop the database and run all query again for the last time

Reply:
I have imported the tables again and I still have 891 as the total number of data, but few data are missing in the cabin from trips tables and age from passengers table. i suppose that is not abnormal.
and while I use passengerId I get 0 as the answer and when I use 'id' I get  7 as the answer.

@adaugochi